client/src/components/App.jsx

```diff
@@ -41,26 +40,54 @@ class App extends React.Component {
         recent_negative_count: null,
         recent_positive_count: null
       },
-      tags: ['Free to play', 'MOBA', 'Strategy', 'Multiplayer', 'PVP']
+      tags: ['Free to play', 'MOBA', 'Strategy', 'Multiplayer', 'PVP'],
```

Comment:
These will need to be generated from API call

Reply:
Sorry I forgot that.